charts/consul/values.yaml

```diff
@@ -1876,6 +1875,83 @@ connectInject:
     # @type: integer
     maxUnavailable: null
 
+  # Configures consul-cni plugin for Consul Service mesh services
+  cni:
+    # If true, then all traffic redirection setup will use the consul-cni plugin. 
+    # Requires connectInject.enabled to also be true.
+    # @type: boolean
+    enabled: false
+ 
+    # If multus plugin is enabled or not. If true, the CNI installer will generate a consul CNI configuration
+    # in a location for the multus program to use. Also, the helm chart will create a custom resource for
+    # NetworkAttachmentDefintion that multus will consume. The values for cniBinDir and cniNetDir must also be 
+    # changed for multus to work. There is a GKE example for cniBinDir below. 
```

Comment:
I think one thing that's still confusing from the example is whether the example is for multus or for GKE or maybe both?

Reply:
You are correct. I removed the GKE reference as it doesn't make sense. Configuration on other platforms and for plugins will need it's own docs as they will be too big to fit in a values file.